services/api/src/typeDefs.js

```diff
@@ -2378,19 +2381,19 @@ const typeDefs = gql`
     addWorkflow(input: AddWorkflowInput!): Workflow
     updateWorkflow(input: UpdateWorkflowInput): Workflow
     deleteWorkflow(input: DeleteWorkflowInput!): String
-    taskDrushArchiveDump(environment: Int!): Task
-    taskDrushSqlDump(environment: Int!): Task
-    taskDrushCacheClear(environment: Int!): Task
-    taskDrushCron(environment: Int!): Task
+    taskDrushArchiveDump(environment: Int!): Task @deprecated(reason: "This task will be removed in a future release. See https://github.com/uselagoon/lagoon/blob/main/DEPRECATIONS.md for alternatives if you use it")
```

Comment:
also need to update `DEPRECATIONS.md` now that 2.17 has released. Maybe something like this
```
### Lagoon v2.18.0
release link: https://github.com/uselagoon/lagoon/releases/tag/v2.18.0
* The standard drupal based tasks that Lagoon ships with (drush ....) have been flagged as deprecated and should not be used anymore. If you require these tasks, see [link to some notes on how to replace them]
```

We should probably write some documentation on how to replace the standard tasks with more Lagoon specific replacements, rather than direct drush replacements?